hdf5-derive/tests/compile-fail/empty-struct.stderr

```diff
@@ -1,7 +1,5 @@
-error: proc-macro derive panicked
- --> $DIR/empty-struct.rs:4:10
+error: Cannot derive `H5Type` for empty structs
```

Comment:
minor, for consistency sake, can we make all error messages (that don't start with a `) start with lowercase letter?

e.g. `cannot derive ...`

Reply:
Fixed